app/views/ile_query.scala.html

```diff
@@ -15,26 +15,53 @@
  *@
 
 @import views.Title
+@import views.html.components.gds.link
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import components.gds.{errorSummary, gds_main_template, pageTitle}
 
-@this(main_template: views.html.templates.main_template)
+@this(
+        govukLayout: gds_main_template,
+        govukButton: GovukButton,
+        govukFieldset: GovukFieldset,
+        govukInput: GovukInput,
+        errorSummary: errorSummary,
+        pageTitle: pageTitle,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(form: Form[String])(implicit request: Request[_], messages: Messages)
 
-@main_template(title = Title("ileQuery.title")) {
+@html = {
+@govukInput(Input(
+    id = "find-consignment",
+    name = "ucr",
+    classes = "govuk-!-width-two-thirds"
+))
+}
 
-  @helper.form(controllers.ileQuery.routes.IleQueryController.submitQueryForm, 'autoComplete -> "off") {
+@govukLayout(
+    title = Title("ileQuery.title")) {
 
-    @helper.CSRF.formField
+    @formHelper(action = controllers.ileQuery.routes.IleQueryController.submitQueryForm(), 'autoComplete -> "off") {
 
-    @components.error_summary(form.errors)
+        @errorSummary(form.errors)
 
-    @components.page_title(Some(messages("ileQuery.title")))
+        @govukFieldset(Fieldset(
+            legend = Some(Legend(
+                content = Text(messages("ileQuery.title")),
+                classes = "govuk-fieldset__legend--xl",
+                isPageHeading = true
+            )),
+            html = html
+        ))
```

Comment:
Please use components where you can. If I remember correctly in this situation you could use `exportsInputText`

Reply:
I agree, but in this case, the component you mentioned is slightly different as `exportsInputText ` does not render the title as in the prototype. Here is a comparison:

![Screen Shot 2020-02-11 at 15 19 48](https://user-images.githubusercontent.com/992887/74250366-65a73c00-4ce2-11ea-9702-20362df65625.png)

Reply:
I will refactor the component to add a class.